SimCalorimetry/Configuration/python/SimCalorimetry_EventContent_cff.py

```diff
@@ -29,5 +28,13 @@
 #
 # Add extra event content if running in Run 2
 #
-eras.run2_common.toModify( SimCalorimetryFEVTDEBUG.outputCommands, func=lambda outputCommands: outputCommands.append('keep *_simHcalUnsuppressedDigis_*_*') )
-eras.run2_common.toModify( SimCalorimetryRAW.outputCommands, func=lambda outputCommands: outputCommands.append('keep *_simHcalUnsuppressedDigis_*_*') )
+from Configuration.Eras.Modifier_run2_common_cff import run2_common
+run2_common.toModify( SimCalorimetryFEVTDEBUG.outputCommands, func=lambda outputCommands: outputCommands.append('keep *_simHcalUnsuppressedDigis_*_*') )
+run2_common.toModify( SimCalorimetryRAW.outputCommands, func=lambda outputCommands: outputCommands.append('keep *_simHcalUnsuppressedDigis_*_*') )
+
```

Comment:
@kpedro88
Can we also add "EBDigiCollection_simHcalUnsuppressedDigis_*_*" here, needed for re-running simEcalEBTriggerPrimitiveDigis ?

<pre>
outputCommands.append('keep EBDigiCollection_simEcalUnsuppressedDigis_*_*') )
</pre>